### PR TITLE
Positive and negative

### PR DESCRIPTION
**Pull request by RogerAlvarezM**

### CODE REVIEW

File: features/project/positive/positive _accounts.feature [hunk 1]
```diff
@@ -0,0 +1,37 @@
+@positive1
```
Comment (estigmata) [inline features/project/positive/positive _accounts.feature]: positive1?
You file name have an space, remove it please

**Review by estigmata — CHANGES_REQUESTED** [review]

File: features/project/positive/positive _accounts.feature (hunk 1 above)
Comment (RogerAlvarezM) [reply features/project/positive/positive _accounts.feature]: done

Comment (estigmata) [inline features/project/positive/positive _accounts.feature]: But I still see an space on file name: positive  _accounts.feature, this name should be posivite.accounts.feature
And ... are you sure this tag is correctly? @positive1?

**Review by estigmata — CHANGES_REQUESTED** [review]

File: features/project/positive/positive _accounts.feature (hunk 1 above)
Comment (estigmata) [reply features/project/positive/positive _accounts.feature]: Can you see the blank space between word "positive" and "_" lower dash?

**Review by estigmata — APPROVED** [review]

**Pull request merged**